File: backend/src/main/java/com/buysell/modules/inventory/dto/BrandSummaryDto.java
package com.buysell.modules.inventory.dto;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class BrandSummaryDto {
    private String brand;
    private Long count;
    private BigDecimal totalValue;
}

File: android/app/src/main/java/com/mobile/app/data/remote/dto/inventory/BrandSummaryDto.kt
package com.mobile.app.data.remote.dto.inventory

import com.google.gson.annotations.SerializedName
import java.math.BigDecimal

data class BrandSummaryDto(
    @SerializedName("brand") val brand: String?,
    @SerializedName("count") val count: Long?,
    @SerializedName("totalValue") val totalValue: BigDecimal?
)
